boards/arm/nucleo_f070rb/nucleo_f070rb.dts

```diff
@@ -107,3 +108,13 @@
 &iwdg {
 	status = "okay";
 };
+
+&adc1 {
+	pinctrl-0 = <&adc_in0_pa0>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&die_temp {
+	status = "okay";
```

Comment:
shouldn't ntc property declared here? Please explain to me... :)

Reply:
ntc is declared in dts/arm/st/f0/stm32f030.dtsi (which is included in nucleo_f070rb), where we declare the entire node.
In nucleo_f070rb, we only enable it.